### PR TITLE
Use standard placeholder names for integration with docs

### PR DESCRIPTION
**Pull request by sereanaseim**
These samples will be embedded in the Dataplex documentation. This change enables the samples on the doc site to display editable placeholder text.

Placeholders can use capital letters, numbers, or underscores. The placeholder text must be different from non-placeholder strings that use all caps.

### CODE REVIEW

File: managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    "IMPORT_JOB_SCOPE_ENTRY_TYPES": [ENTRY_TYPES],
+    "IMPORT_JOB_SCOPE_ASPECT_TYPES": [ASPECT_TYPES],
     "IMPORT_JOB_LOG_LEVEL": "INFO",
-    "NETWORK_TAGS": [],
-    "NETWORK_URI": "",
-    "SUBNETWORK_URI": ""
- }
\ No newline at end of file
+    "NETWORK_TAGS": [TAGS],
+    "NETWORK_URI": "NETWORK",
```
Comment (vinitachaudhary) [inline managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json]: Leave network_uri and subnetwork_uri empty.

Comment (sereanaseim) [reply managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json]: Done. Also removed them from the .tfvars sample.

What about NETWORK_TAGS, should I leave that empty too?

File: managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-    "CLOUD_REGION": <REGION_ID>,
-    "TARGET_ENTRY_GROUP_ID": <ENTRY_GROUP_ID>,
-    "CREATE_TARGET_ENTRY_GROUP": <true/false>,
-    "CLOUD_STORAGE_BUCKET_ID": <BUCKET_ID>,
-    "SERVICE_ACCOUNT": <SERVICE_ACCOUNT_ID>,
-    "ADDITIONAL_CONNECTOR_ARGS": [],
-    "CUSTOM_CONTAINER_IMAGE": <IMAGE>,
-    "IMPORT_JOB_SCOPE_ENTRY_TYPES": [],
-    "IMPORT_JOB_SCOPE_ASPECT_TYPES": [],
+    "TARGET_PROJECT_ID": PROJECT_ID,
```
Comment (vinitachaudhary) [inline managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json]: All the arguments here still need to be in double quotes, except for CREATE_ENTRY_GROUP_BOOLEAN.

Comment (sereanaseim) [reply managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json]: Done. Good catch.

File: managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json (hunk 1 above)
Comment (vinitachaudhary) [reply managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json]: yes

File: managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-    "IMPORT_JOB_SCOPE_ASPECT_TYPES": [],
+    "TARGET_PROJECT_ID": "PROJECT_ID",
+    "CLOUD_REGION": "LOCATION_ID",
+    "TARGET_ENTRY_GROUP_ID": "ENTRY_GROUP_ID",
+    "CREATE_TARGET_ENTRY_GROUP": CREATE_ENTRY_GROUP_BOOLEAN,
+    "CLOUD_STORAGE_BUCKET_ID": "BUCKET_ID",
+    "SERVICE_ACCOUNT": "SERVICE_ACCOUNT_ID",
+    "ADDITIONAL_CONNECTOR_ARGS": ["ADDITIONAL_CONNECTOR_ARGUMENTS"],
+    "CUSTOM_CONTAINER_IMAGE": "CONTAINER_IMAGE",
+    "IMPORT_JOB_SCOPE_ENTRY_TYPES": ["ENTRY_TYPES"],
```
Comment (vinitachaudhary) [inline managed-connectivity/cloud-workflows/byo-connector/args/byo-connector-workflow-args.json]: ENTRY_TYPES, ASPECT_TYPES and ADDITIONAL_CONNECTOR_ARGUMENTS are meant to be list of strings, so they shouldn't be in double quotes.

**Review by vinitachaudhary — APPROVED** [review]

**Pull request merged**